CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Minimal supported Scarb version is now `2.9.1`
 
+### Cast
+
+### Fixed
+
+- User is now prompted to save an imported or deployed account in `sncast` config even when using `--network` flag
```

Comment:
Why do we actually mention `--network` flag here?

Reply:
I can remove it, the bug was due to sncast checking if `--network` flag was passed to determine if to show the prompt.

Reply:
Ok I see 👍